OPAL/br/src/main/scala/org/opalj/br/fpcf/analyses/immutability/ClassImmutabilityAnalysis.scala

```diff
@@ -193,12 +169,10 @@ class ClassImmutabilityAnalysis(val project: SomeProject) extends FPCFAnalysis {
         }
     }
 
-    private[this] object SuperClassKey
```

Comment:
Question to check if understand this correctly: Using this singleton previously only allowed one type of dependency on super class information?

Reply:
I think so, but it seems if that was ever really necessary, it was in an earlier version with a different analysis design. When I removed it, the analysis broke, but only because wrong dependencies were registered.